app/helpers/xls.py

```diff
@@ -220,10 +221,21 @@ def get_columns_in_main_sheet(require_expenditures, require_mission_name):
                 13,
                 light_green_hex,
             ),
+            (
+                "Temps de liaison",
```

Comment:
In other columns we don't specify "Temps de", shouldn't we put only `"Liaison"` here to be consistent?